app/experimenter/experiments/constants/nimbus.py

```diff
@@ -64,6 +67,7 @@ class ApplicationConfig:
         Channel.RELEASE: "org.mozilla.ios.Firefox",
     },
     default_app_id="",
+    rs_experiments_collection="nimbus-mobile-experiments",
```

Comment:
We have the collection name constants in settings:

https://github.com/mozilla/experimenter/blob/main/app/experimenter/settings.py#L416-L418

which is already imported in this file:

https://github.com/mozilla/experimenter/blob/d0aef25a9a261276098d5225c7d317a3b330295e/app/experimenter/experiments/constants/nimbus.py#L4

Reply:
yeah but you want it to be associated with the application, no?

Reply:
or do you mean just like `rs_experiments_collection=settings.whatever`